### ISSUE TITLE
Incorrect aria label on datepicker buttons

### ISSUE THREAD

**Issue by cvillella** [primefaces/primeng#18405]
### Describe the bug

All buttons should have an accessible name, in the datepicker component, several of the aria-labels are placed directly onto the p-button tag instead of being passed into the component to be placed on the actual button element.

### Pull Request Link

https://github.com/primefaces/primeng/pull/18404

### Reason for not contributing a PR

- [ ] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://primeng.org/datepicker#time

### Environment

Can be reproduced on any environment running Angular 19  with PrimeNG 19.

### Angular version

19

### PrimeNG version

v19

### Steps to reproduce the behavior

Inspect .p-datepicker-increment-button within https://primeng.org/datepicker#time datepicker overlay panel and noticed the button tags don't have aria-label set, instead the p-button tag wrapping it has the aria-label

### Expected behavior

All .p-datepicker-increment-button have an accessible name on them (they are icon only buttons which require aria-label be set to pass accessibility)

**Comment by Platacoder** [comment]
This is still an issue!
